tests/unit/s3/test_lifecycle.py

```diff
@@ -50,27 +50,90 @@ def default_body(self):
             <Status>Disabled</Status>
             <Transition>
               <Date>2012-12-31T00:00:000Z</Date>
-              <StorageClass>GLACIER</StorageClass>
+              <StorageClass>STANDARD_IA</StorageClass>
+            </Transition>
+            <Expiration>
+              <Date>2012-12-31T00:00:000Z</Date>
+            </Expiration>
+          </Rule>
+          <Rule>
+            <ID>multiple-transitions</ID>
+            <Prefix></Prefix>
+            <Status>Enabled</Status>
+            <Transition>
+                <Days>30</Days>
+                <StorageClass>STANDARD_IA</StorageClass>
+            </Transition>
+            <Transition>
+                <Days>90</Days>
+                <StorageClass>GLACIER</StorageClass>
             </Transition>
           </Rule>
         </LifecycleConfiguration>
         """
 
-    def test_parse_lifecycle_response(self):
+    def _get_bucket_lifecycle_config(self):
         self.set_http_response(status_code=200)
         bucket = Bucket(self.service_connection, 'mybucket')
-        response = bucket.get_lifecycle_config()
-        self.assertEqual(len(response), 2)
-        rule = response[0]
+        return bucket.get_lifecycle_config()
+
+    def test_lifecycle_response_contains_all_rules(self):
+        self.assertEqual(len(self._get_bucket_lifecycle_config()), 3)
+
+    def test_parse_lifecycle_id(self):
+        rule = self._get_bucket_lifecycle_config()[0]
         self.assertEqual(rule.id, 'rule-1')
+
+    def test_parse_lifecycle_prefix(self):
+        rule = self._get_bucket_lifecycle_config()[0]
         self.assertEqual(rule.prefix, 'prefix/foo')
+
+    def test_parse_lifecycle_no_prefix(self):
+        rule = self._get_bucket_lifecycle_config()[2]
+        self.assertEquals(rule.prefix, '')
+
+    def test_parse_lifecycle_enabled(self):
+        rule = self._get_bucket_lifecycle_config()[0]
         self.assertEqual(rule.status, 'Enabled')
+
+    def test_parse_lifecycle_disabled(self):
+        rule = self._get_bucket_lifecycle_config()[1]
+        self.assertEqual(rule.status, 'Disabled')
+
+    def test_parse_expiration_days(self):
+        rule = self._get_bucket_lifecycle_config()[0]
         self.assertEqual(rule.expiration.days, 365)
-        self.assertIsNone(rule.expiration.date)
-        transition = rule.transition
-        self.assertEqual(transition.days, 30)
```

Comment:
Seems like this behavior would no longer work, correct?  Because `transition` is now a list, instead of `rule.transition.days`, you'd have to say `rule.transition[0].days`.